pages/events/index.vue

```diff
@@ -1,5 +1,4 @@
 <script setup>
-import { addURLParams, removeURLParams } from "@/components/utils/urlUtils";
 import { events } from "@/content/events.json";
 
 useHead({
```

Comment:
Looks like `eventsTags` is not in real use, and also it probably should not be a computed as it does not change over time.